config/credentials/development.yml.enc

```diff
@@ -0,0 +1 @@
+5luXbLYh08SXUF+vPuqZSPyLmJBFmfTOXw8pMCij3W+BhfOUIhP53uBWBYsqkip/yB/GVPEI1vko3YrtCDLheZxuac+BUaD1dIsluPbhqeHVDohMYkZwQCSA24iqs1QV4co0Ase8VEo6AmzGv9haunXAehJKsYYQWd8jdOTj8UPgO48T8Ji/8U0zlWQ+iCvxlohzCQuaXHMWYrGOUJztdcvXUIy78Dzlb5lyHQ5n69EuTNDTBMW0w/4eJKvRNhq2mNG5iBR7Htqh5txpvoPDG/vW6wVEhWAat5NtYPuE+J9vznXZ0MTAFkE5ksSvUb9ygbCXEsWpHePAoH8=--spr8lyj9JJuglCr5--WEKKqmx8qVFwT9ZT9dgsMQ==
```

Comment:
I actually don't think we need a development credentials file for this app. Rails has automatic `secret_key_base` generation logic for dev/test environments and the google maps api key for dev is bullshit anyways